### PR TITLE
Flo ne list extractor

### PR DESCRIPTION
**Pull request by floschne**
See https://mafiasi.de/etherpad/pad/nlpFloJulian/NLP_NER_Project

### CODE REVIEW

File: src/main/resources/ner/dataSources.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+http://www.countries-list.info/
+
+http://www.adrianworlddesign.com/downloads/usStates.txt
+
+wikipedia
```
Comment (julianbetz) [inline src/main/resources/ner/dataSources.txt:5]: Which of the data are from wikipedia? Do you still have the link + date when you looked up the data?

File: src/main/java/de/unihamburg/informatik/nlp4web/tutorial/tut5/feature/NEListExtractor.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    private void generateNeToTagMap(File nerListFile) throws IOException {
+        try (BufferedReader reader = new BufferedReader(new FileReader(nerListFile))) {
+            String line;
+            StringBuilder sb = new StringBuilder();
+            // read line by line
+            while ((line = reader.readLine()) != null) {
+                String[] split = line.split(delimiterSymbol);
+                // since a name can contain more than two space-separated strings, the splitted strings are combined again
+                for (int i = 1; i < split.length; ++i)
+                    sb.append(split[i]).append(" ");
```
Comment (julianbetz) [inline src/main/java/de/unihamburg/informatik/nlp4web/tutorial/tut5/feature/NEListExtractor.java]: The data we train on is tokenized to form single words only (i.e. no spaces in the tokens). Is it really helpful to save the concatenation of names in the map, and then search only for single words in the map later on? Or am I missing something?

**Review by julianbetz — APPROVED** [review]
All in all fine. I commented two sections that raised some questions for me.

File: src/main/java/de/unihamburg/informatik/nlp4web/tutorial/tut5/feature/NEListExtractor.java (hunk 2 above)
Comment (floschne) [reply src/main/java/de/unihamburg/informatik/nlp4web/tutorial/tut5/feature/NEListExtractor.java]: Let's have a talk tomorrow about that :) ?

Comment (julianbetz) [reply src/main/java/de/unihamburg/informatik/nlp4web/tutorial/tut5/feature/NEListExtractor.java]: Sure, but maybe on Wednesday or Thursday, that would fit my schedule better :)

**Pull request merged**